models/reg_lstm/__main__.py

```diff
@@ -147,22 +156,23 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
 
     if not args.trained_model:
         trainer.train(args.epochs)
+        model = torch.load(trainer.snapshot_path)
     else:
         if args.cuda:
             model = torch.load(args.trained_model, map_location=lambda storage, location: storage.cuda(args.gpu))
         else:
             model = torch.load(args.trained_model, map_location=lambda storage, location: storage)
 
-    model = torch.load(trainer.snapshot_path)
```

Comment:
This doesn't work if `trained-model` is specified because it
* Overwrites the line above (`torch.load(args.trained_model...`)
* Tries to load a model whose file name is the current timestamp, which may not exist